modules/comment-likes/comment-like-count.js

```diff
@@ -0,0 +1,55 @@
+jQuery( document ).ready( function( $ ) {
+	var jsonAPIbase = 'https://public-api.wordpress.com/rest/v1',
+		APIqueue = [];
+
+	function getCommentLikeCounts() {
+		$( '.comment-like-count' ).each( function() {
+			var blogId = $( this ).attr( 'data-blog-id' ),
+				commentId = $( this ).attr( 'data-comment-id' );
+
+			APIqueue.push( '/sites/' + blogId + '/comments/' + commentId + '/likes' );
+		} );
+
+		fetchCounts();
+	}
+
+	function showCount( commentId, count ) {
+		if ( count < 1 ) {
+			return;
+		}
+
+		$( '#comment-like-count-' + commentId ).find( '.like-count' ).hide().text( count ).fadeIn();
+	}
+
+	function fetchCounts() {
+		var batchRequest = {
+			path: '/batch',
+			data: 'urls[]=' + APIqueue.join( '&urls[]=' ),
```

Comment:
Yes. Just for future reference @georgestephanis already did this in https://github.com/Automattic/jetpack/pull/7426.